modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -125,6 +126,16 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
       storages.block.store(fb.hashedBlock).handleErrorWith(e => logger.debug(e)("Block storing failed."))
     }
 
+  private val sendBlockToL0: Pipe[F, FinalBlock, Unit] =
+    _.evalMap { fb =>
+      (for {
+        tips <- storages.block
+          .pullTips(appConfig.tips.maximumTipsCount)
```

Comment:
Maybe we should only send the lowest tip height? All depends on how this information will be used on the L0.